From 2ee98c517b7ba4a130af7cce7677bd488fc5712d Mon Sep 17 00:00:00 2001
From: Drew DeVault <sir@cmpwn.com>
Date: Fri, 22 Dec 2017 11:38:38 -0800
Subject: [PATCH] Import sleep nearer to usage

The version imported at the top is overwritten later, breaking tests.
---
 tqdm/tests/tests_tqdm.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tqdm/tests/tests_tqdm.py b/tqdm/tests/tests_tqdm.py
index d668248..ac0948b 100644
--- a/tqdm/tests/tests_tqdm.py
+++ b/tqdm/tests/tests_tqdm.py
@@ -89,6 +89,7 @@ class FakeSleep(object):
         self.dtimer = dtimer
 
     def sleep(self, t):
+        from time import sleep
         end = t + self.dtimer.t
         while self.dtimer.t < end:
             sleep(0.0000001)  # sleep a bit to interrupt (instead of pass)
-- 
2.15.0

